commands/describe.go

```diff
@@ -103,20 +104,11 @@ func runDescribe(cmd *cobra.Command, args []string) error {
 	url, asyncURL := getFunctionURLs(gatewayAddress, functionName, functionNamespace)
 
 	funcDesc := schema.FunctionDescription{
```

Comment:
Why were these fields deleted?

Reply:
Because i just embed the FunctionStatus now instead of copying each field one-at-a-time. This will ensure that the FunctionDescription always has access to the FunctionStatus fields

Reply:
see https://github.com/openfaas/faas-cli/pull/915/files/20f8af79081c4b3e7d175804609fc4ebd541ca54#diff-cc6e60246691f8b3534c70829a1eaa3a123849240f9c046c2b46f5abf917be2cR10

Reply:
Thanks for explaining.